callable.sh

```diff
@@ -24,6 +24,13 @@ Github__callable_help(){
 # @param $1: The label config file
 ##################################################
 Github__callable_labels(){
+    # Checking if variable exists
+    Github_validate_token
+    if [[ $? -ne 0 ]]; then
```

Comment:
@kylemac I believe this will be the only part you could get tripped up on.

In bash when a function actually `return`s as opposed to "echo" how I usually do it, you grab the return value of the last function through `$?`